react-basic-practice/src/components/ListProducts/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import Grid from '@/layouts/Grid';
+import ProductCard from '../ProductCard';
+import { getListProductsLimit } from '../../services/Products';
+
+const ListProducts = () => {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    const getData = async () => {
```

Comment:
Redundant.

Reply:
because async await cannot be used directly in a component or useEffect, so a function is needed to use it